app/assets/stylesheets/wiki.css

```diff
@@ -83,3 +83,15 @@ a.navbar-offset {
   background: white;
 }
 
+.modal{
```

Comment:
Aha! it looks like some custom styles. I wonder though if we can just use the standard "Bootstrap" styles as described at http://getbootstrap.com/docs/4.4/components/modal/ -- would you be able to make that change? 

And, would you be able to share a screenshot of what it looks like with your work added? Thank you so much!